python/tvm/tir/schedule/schedule.py

```diff
@@ -1303,6 +1304,11 @@ def compute_at(
         preserve_unit_loops: bool
             Whether to keep the trivial loops whose extents are 1
 
+        index: int
+            The block index of the loop body subtree blocks
+            -1 means inserted into the last possible insertion point
+            -2 means inserted into the first possible insertion point
```

Comment:
```suggestion
            The block index of the loop body subtree blocks:
            - `index = -1` means inserted into the last possible insertion point;
            - `index = -2` means inserted into the first possible insertion point;
            - Otherwise, `index` is a nonnegative number that indicates the insertion point
```